integrationTests/__snapshots__/failing.test.js.snap

```diff
@@ -6,7 +6,7 @@ exports[`Works when it has failing tests 1`] = `
   4:3  error  'hello' is not defined  no-undef
   5:3  error  'bye' is not defined    no-undef
 ✖ 2 problems (2 errors, 0 warnings)
-  ✕ ESLint
+  ✓ ESLint
```

Comment:
not sure about this one... @rogeliog thoughts?

Reply:
This should definitely be a blocker; the check shouldn’t appear if there are any failures.

Reply:
yup. it was a bug in my port, fixed now 🙂